.prospector.yaml

```diff
@@ -20,6 +20,7 @@ ignore-paths:
   - pip-wheel-metadata
   - lispcore.py
   - tests
+  - venv
```

Comment:
Am I correct in understanding this is related to the PR description of #705?

Just to dig in a bit, I'm a little curious why you have a local venv folder rather than letting Poetry handle it?

Reply:
I have multiple environments so I can test the different python versions, 3.8 to 3.11. I'm new to `Poetry` and `tox` and haven't looked into them in detail yet. I'll shall follow up in #705. Thanks